.github/workflows/test.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       - name: docker login
         run: docker login https://docker.pkg.github.com -u SwedbankPay --password ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Cache HTMLProofer
+        id: cache-htmlproofer
+        uses: actions/cache@v2
+        with:
+          path: tmp/.htmlproofer
+          key: ${{ runner.os }}-htmlproofer
+
       - name: build
```

Comment:
I think this needs to be moved to the `verify` job, somewhere before `name: Verify build` on line 53. :)